src/main/java/net/sf/jabref/exporter/MSBibExportFormat.java

```diff
@@ -61,13 +61,9 @@ public void performExport(final BibDatabase database, final MetaData metaData, f
             } catch (Exception e) {
                 throw new Error(e);
             }
-        }
-        try {
             finalizeSaveSession(ss);
-        } catch (SaveException ex) {
+        } catch (Exception ex) {
```

Comment:
Sure. However, as the same thing was done both for IOException and Exception, it seemed to make more sense to only catch Exception (which, of course, is bad in itself). Not sure why the code looks like it does and which exceptions make sense to catch.